modules/rhel-preparing-playbook-machine.adoc

```diff
@@ -7,7 +7,12 @@
 [id="rhel-preparing-playbook-machine_{context}"]
 = Preparing the machine to run the playbook
 
-Before you can add compute machines that use {op-system-base-full} as the operating system to an {product-title} {product-version} cluster, you must prepare a {op-system-base} 7 machine to run an Ansible playbook that adds the new node to the cluster. This machine is not part of the cluster but must be able to access it.
+Before you can add compute machines that use {op-system-base-full} as the operating system to an {product-title} {product-version} cluster, you must prepare a {op-system-base} 7 or 8 machine to run an Ansible playbook that adds the new node to the cluster. This machine is not part of the cluster but must be able to access it.
++
```

Comment:
This is not rendering correctly.
![A superfluous plus sign floating in the middle of a documentation page](https://user-images.githubusercontent.com/50179998/179808002-1cc3eba8-bb52-4704-b905-46538e64f92d.png)